test/monitoring/aws-rds/AuroraClusterMonitoring.test.ts

```diff
@@ -31,58 +25,6 @@ test("snapshot test: no alarms", () => {
   expect(Template.fromStack(stack)).toMatchSnapshot();
 });
 
-test("RDS passed into Aurora Cluster", () => {
```

Comment:
Was `DatabaseCluster` removed? Trying to follow why we don't want this test anymore

Reply:
Ah it was buried in the commit message description:

> This unfortunately drops the guard for non-serverless DatabaseClusters for Aurora monitoring since the constructs are now nearly indistinguishable based on their properties.